packages/commerce-events-sdk/tests/contexts.test.ts

```diff
@@ -20,6 +20,8 @@ import {
     generateStorefrontInstanceContext,
 } from "./mocks";
 
+const mdl = new MagentoStorefrontEvents();
+
```

Comment:
should this be `mse`?

Reply:
it probably should be but the rest of the tests were using `mdl` as the variable name